### ISSUE TITLE
Recover partitioned deleted Veracrypt Partition

### ISSUE THREAD

**Issue by kikieri** [veracrypt/VeraCrypt#790]
Hey Guys,
I had an encrypted Veracrypt Partition mounted from an external Storage and went to my Filemanager on Macosx when I had an other USB Drive mounted and accidentaly formated my Ecnrypted Partition to FAT32 empty. Is there any way to recover my files. I didnt touch anything afterwards.

**Comment by viciousvex** [comment]
Yes. There should be a way, because a FAT32 partition does not overwrite the end of the volume (which contains a backup VeraCrypt header). Look here for instructions on how to restore the backup header: https://veracrypt.fr/en/Program%20Menu.html#tools-restore-volume-header

**Comment by kikieri** [comment]
I recovered my Volume Header. I can mount the drive. But still it just show the formated FAT drive again. Data of the osx should be still there. What to do?

**Comment by karakuraizer** [comment]
> I recovered my Volume Header. I can mount the drive. But still it just show the formated FAT drive again. Data of the osx should be still there. What to do?

now when you mounted your encrypted recovered partition you can try to use any software made for scanning FS for any deleted data, wish you a good luck=)
